webhook_server_container/libs/github_api.py

```diff
@@ -2071,3 +2058,41 @@ def run_podman_command(self, command: str, pipe: bool = False) -> Tuple[bool, st
             return run_command(command=command, log_prefix=self.log_prefix, pipe=pipe)
 
         return rc, out, err
+
+    def get_approvers_and_reviewers(self) -> dict[str, dict[str, list[str]]]:
+        # Dictionary mapping OWNERS file paths to their approvers and reviewers
+        _owners: dict[str, dict[str, list[str]]] = {}
+
+        max_owners_files = 1000  # Configurable limit
```

Comment:
You can limit with folder level too. That way you can limit what a SIG can be (upto level 3 - root>folder level1> folderlevel 2) etc.

Reply:
ANy reason to limit this?

Reply:
Data on SIG is all over the place. Initially based on what I was reading, I was thinking a SIG has to be large enough to justify forming it. If that is true then level 3/4 would be good enough. But then I found staging/src/kubevirt.io/api/OWNERS and while it is one instance, since this repo is not just managing one repo, you can't predict or restrict levels. So taking that comment back.